streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStreamThread.java

```diff
@@ -294,7 +298,7 @@ public void run() {
 
     private StateConsumer initialize() {
         try {
-            final GlobalStateManager stateMgr = new GlobalStateManagerImpl(topology, consumer, stateDirectory);
+            final GlobalStateManager stateMgr = new GlobalStateManagerImpl(topology, consumer, stateDirectory, stateRestoreListener);
```

Comment:
nit: should probably put the params on separate lines as it has exceeded the 120 char limit

Reply:
Ack